src/NetAnalyzers/Microsoft.CodeAnalysis.NetAnalyzers.md

```diff
@@ -1452,6 +1452,18 @@ For improved performance, use the LoggerMessage delegates.
 |CodeFix|False|
 ---
 
+## [CA1849](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1849): Call async methods when in an async method
+
+When inside a Task-returning method, use the async version of methods, if they exist.
+
+|Item|Value|
+|-|-|
+|Category|Performance|
+|Enabled|False|
+|Severity|Warning|
```

Comment:
Hm, this is strange, i did not update it manually `msbuild /t:pack` updated this, but i guess Severity should no be as a Warning for disabled analyzers, should I update the severity manually? @mavasani @jmarolf

Reply:
oh i see all other `Disabled` analyzers has same value, guess it is expected then